blockchain/types/tx_internal_data.go

```diff
@@ -697,6 +697,64 @@ func IntrinsicGas(data []byte, accessList AccessList, authorizationList Authoriz
 	return gasPayloadWithGas, tokens, nil
 }
 
+func GetTxGasForTxType(txType TxType) uint64 {
```

Comment:
Maybe we can introduce a predefined map from types to gas. What do you think?

```
var txTypeToGasMap map[TxType]uint64{
    TxTypeLegacyTransaction: params.TxGas,
    ...
}

func GetTxGasForTxType(txType TxType) (uint64, error) {
    if gas, exists := txTypeGasTable[txType]; exists {
	return gas, nil
    }
    return 0, errors.New(...)
```

Reply:
Done, 6d4eea0